templates/register.html

```diff
@@ -1,64 +1,101 @@
-        </div>
-
-</body>
-
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="/static/base.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" type="text/css" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+    <style>
+    	.icon-wrapper{
+			
+			display: inline;
+			float: right;
+			
+			}
+
+			.ion-eye,
+			.ion-more{
+				
+				position: relative;
+				top: -25px;
+				float: right;
+				right: 20px;
+				cursor: pointer;
+				}
+			@media only screen and (max-width:768px) {
+            .img-fluid{
+                display: none;
+            }
+        }	
+		
+    </style>
+
+</head>
+
+<body>
+
+    <nav class="navbar navbar-expand-md bg-dark navbar-dark">
+        <a class="navbar-brand" href="/home">CS50</a>
+        <ul class="nav navbar-nav  ">
+            <li class="nav-item">
+                <a class="nav-link" href="/login">Login</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="/register">Register</a>
+            </li>
+
+        </ul>
+
+    </nav>
+
+    <div class="container">
+        <h2>    Register </h2>
+        <br>
+        <div class="row">
+
+            <div class="col-5">
+                <form action="" method="post">
+
+                    <div class="form-group">
+                        <label for="exampleInputEmail1">Name</label>
+                        <input type="text" name="name" class="form-control" placeholder="Enter name">
+
+                    </div>
+                    <div class="form-group">
+                        <label for="exampleInputEmail1">Email address</label>
+                        <input type="email" name="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
+
+                    </div>
+                    <div class="form-group">
+                        <label for="exampleInputPassword1">Password</label>
+                        <input type="password" name="password" class="form-control icon-wrapper input" id="exampleInputPassword1" placeholder="Password">
+                        <span toggle="#exampleInputPassword1" onclick="myFunction()" class="ion ion-eye field-icon toggle-password"></span>
+                    </div>
+
+                    <button type="submit" class="btn btn-dark">Submit</button>
+                </form>
+
+            </div>
+            <div class="col-7"><img src="/static/registerPg_Finance.png" alt="" width="760" height="450"></div>
+        </div>
+
+        <div class="footer">
+            <p class="text-center"> Finance tracker</p>
+        </div>
+
+</body>
+<script>
+    	function myFunction() {

```

Comment:
And change the function name from **myFunction()** to something very specific and relevant